src/Components/AssignmentDisplay/AssignmentDisplay.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from "react";
+import { GlobalStyle } from "../../Styles/GlobalStyle";
+import Assignment1 from "../../Screens/Assignment1/Assignment1";
+import Assignment2 from "../../Screens/Assignment2/Assignment2";
+import Assignment3 from "../../Screens/Assignment3/Assignment3";
+import AssignmentMenu from "../../Screens/AssignmentMenu/AssignmentMenu";
+
+const AssignmentDisplay = () => {
+  const [activeScreen, setActiveScreen] = useState(0);
+
+  const handleMenuSelection = (selectedScreen: number) => {
+    setActiveScreen(selectedScreen);
+  };
+
+  return (
+    <>
+      <GlobalStyle />
+      {activeScreen === 0 && (
```

Comment:
Instead of comparing with 0, 1 or 2, can you please use proper names of that screens.